js/page-specific/phone-directory/phone-directory.html

```diff
@@ -133,62 +125,83 @@
+              <option value="County Auditors">County Auditors</option>
+              <option value="County Council">County Council</option>
+              <option value="Custom Logic Design Vendor"
+                >Custom Logic Design Vendor</option
+              >
+              <option value="Economic and Workforce Development"
+                >Economic and Workforce Development</option
+              >
+              <option value="Elections">Elections</option>
+              <option value="Environmental Protection and Sustainability"
+                >Environmental Protection and Sustainability</option
+              >
+              <option value="Executive Office">Executive Office</option>
+              <option value="Fire">Fire</option>
+              <option value="Health">Health</option>
+              <option value="Human Resources">Human Resources</option>
+              <option value="Information Technology"
+                >Information Technology</option
+              >
+              <option value="Law">Law</option>
+              <option value="Liquor Board">Liquor Board</option>
+              <option value="Local Management Board"
+                >Local Management Board</option
+              >
+              <option value="Office of Administrative Hearings"
+                >Office of Administrative Hearings</option
+              >
+              <option value="Office of Housing">Office of Housing</option>
+              <option value="Orphan&#39;s Court">Orphan&#39;s Court</option>
+              <option value="Permits, Approvals and Inspections"
+                >Permits, Approvals and Inspections</option
+              >
+              <option value="Planning">Planning</option>
+              <option value="Police">Police</option>
+              <option value="Property Management">Property Management</option>
+              <option value="Public Works">Public Works</option>
+              <option value="Recreation and Parks">Recreation and Parks</option>
+              <option value="Sheriff&#39;s Office">Sheriff&#39;s Office</option>
+              <option value="Social Services">Social Services</option>
+              <option value="State&#39;s Attorney&#39;s Office"
+                >State&#39;s Attorney&#39;s Office</option
+              >
             </select>
           </div>
           <div class="buttonContain">
-            <input
+            <button
               id="btnSubmit"
               name="submit"
+              class="seButton contact-btn"
               type="submit"
-              value="Submit"
-              tabindex="4"
-            />
-            <input
+            >
+              Submit
+            </button>
+            <button
               id="btnClear"
               name="btnClear"
+              class="seButton contact-btn clear-btn"
               type="button"
-              value="Clear"
-              tabindex="5"
-              class="btnWidths"
```

Comment:
@srutherfordbcg the important part here is the buttons changing from input elements to buttons elements.

Reply:
Looks good - It does seem redundant to have a button object with a type "button" - I wonder if that is still necessary.

Reply:
@srutherfordbcg some light reading on the topic, https://dev.to/clairecodes/why-its-important-to-give-your-html-button-a-type-58k9